suite-common/wallet-core/src/accounts/accountsReducer.ts

```diff
@@ -194,6 +194,18 @@ export const selectAccountByDescriptorAndNetworkSymbol = memoizeWithArgs(
     },
 );
 
+export const selectAccountKeyByDescriptorAndNetworkSymbol = memoizeWithArgs(
+    (state: AccountsRootState, accountDescriptor: string, networkSymbol: NetworkSymbol) => {
+        const account = selectAccountByDescriptorAndNetworkSymbol(
+            state,
+            accountDescriptor,
+            networkSymbol,
+        );
+
+        return account?.key;
```

Comment:
nit: I'd return `null` instead of undefined

Reply:
Also I think it's quite nice to type return value from selectors. Even though ts can see what the return value is based on code changes, it is IMO good to "lock" that value with return types.

Reply:
So should I change it then?

Reply:
@Nodonisko Yeah, it won't change anything, just semantic thing. Feel free to resolve if you disagree

Reply:
return type added in 53e695678